src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -279,15 +282,41 @@ public void ProcessPushNotification(PushNotification pushNotification, TimeSpan?
                     $"{nameof(pushNotification)}.{nameof(pushNotification.ResourceUri)}");
             }
 
-            _client.UpdateSyncToken(pushNotification.SyncToken);
+            if (_configClientProvider.UpdateSyncToken(pushNotification.ResourceUri, pushNotification.SyncToken))
+            {
+                SetDirty(maxDelay);
+            }
+            else
+            {
+                _logger.LogWarning($"Ignoring the push notification received for endpoint '{pushNotification.ResourceUri}' that is not present in the applications' configured list of endpoints.");

```

Comment:
"application" can be a bit confusing what we refer to.

Will this be clearer?
"... not present in the input list of endpoints."

Reply:
I think 'input' makes it clearer.